saleor/core/permissions.py

```diff
@@ -157,3 +157,26 @@ def get_permissions_from_codenames(permission_codenames: List[str]):
         .prefetch_related("content_type")
         .order_by("codename")
     )
+
+
+def _permission_required(perms, context):
+    from saleor.account.models import User
```

Comment:
If this import is here because of circular dependency, you can also try to import the user model using this Django helper:
```
# in the import section:
from django.contrib.auth import get_user_model

# in the function:
model = get_user_model()
```